src/components/loader-spinner/loader-spinner.style.ts

```diff
@@ -0,0 +1,176 @@
+}
+
+export const StyledSpinnerWrapper = styled.div<StyledSpinnerWrapperProps>`
+  ${margin}
+  ${({ size, reduceMotion, color }) =>
+    size &&
+    css`
+      width: ${LOADER_SPINNER_SIZE_PARAMS[size].wrapperDimensions}px;
+      height: ${LOADER_SPINNER_SIZE_PARAMS[size].wrapperDimensions}px;
+      min-width: ${LOADER_SPINNER_SIZE_PARAMS[size].wrapperDimensions}px;
+      min-height: ${LOADER_SPINNER_SIZE_PARAMS[size].wrapperDimensions}px;
+
+      ${!reduceMotion &&
+      css`
+        box-shadow: inset 0px 0px 0px
+          ${LOADER_SPINNER_SIZE_PARAMS[size].boxShadow}px
+          ${calculateColors("Wrapper", color)};
+        border-radius: 50%;
+      `}
+    `}
+`;
+
+export const StyledLabel = styled(Typography)`
+  display: flex;
+  justify-content: center;
+`;
+
+export const StyledSpinnerCircleSvg = styled.svg<
+  Omit<LoaderSpinnerProps, "showSpinnerLabel" | "labelColor">
+>`
+  ${({ size, tracked, motion, animationTime, color }) =>
+    size &&
+    css`
+      x: 0px;
+      y: 0px;
+
+      circle {
+        fill: transparent;
+        stroke-width: ${LOADER_SPINNER_SIZE_PARAMS[size].strokeWidth}px;
+        stroke: ${calculateColors("Wheel", color)}
+        stroke-linecap: round;
+
+        stroke-dasharray: 100px;
+        stroke-dashoffset: 80px;
+
+        transform-origin: 12px 12px 0px;
+
+        cx: 12px;
+        cy: 12px;
+        r: 10px;
+
+        transform: rotate(270deg);
+
+        @keyframes gradientAnimation {
+          0% {
+            transform: rotate(0deg);
+            stroke-dasharray: 100;
+            stroke: #00D639;
+          }
+
+          33% {
+            stroke: #11AFFF;
+            stroke-dasharray: 90;
+          }
+
+          50% {
+            stroke-dasharray: 80;
+          }
+
+          66% {
+            stroke: #8F49FE;
+            stroke-dasharray: 90;
+          }
+
+          100% {
+            transform: rotate(340deg);
+            stroke-dasharray: 100;
+            stroke: #00D639;
+          }
+        }
+
+        @keyframes trackedAnimation {
+          0% {
+            stroke-dasharray: 100;
+            stroke-dashoffset: 100;
+          }
+
+          100% {
+            stroke-dasharray: 100;
+            stroke-dashoffset: 20;
+          }
+        }
+
+        @keyframes untrackedAnimation {
+          0% {
+            transform: rotate(0deg);
+            stroke-dasharray: 100;
+          }
+
+          20% {
+            transform: rotate(70deg);
+            stroke-dasharray: 90;
+          }
+
+          40% {
+            transform: rotate(140deg);
+            stroke-dasharray: 80;
+          }
+
+          60% {
+            transform: rotate(210deg);
+            stroke-dasharray: 90;
+          }
+
+          80% {
+            transform: rotate(280deg);
+            stroke-dasharray: 100;
+          }
+
+          100% {
+            transform: rotate(360deg);
+            stroke-dasharray: 100;
+          }
+        }
+
```

Comment:
**suggestion**: what do you think about reducing the number of keyframes for each of the three animations, in order to improve performance slightly?:

```suggestion
        @keyframes gradientAnimation {
          0% {
            stroke: #00D639;
          }

          33% { 
            stroke: #11AFFF;
          }

          66% {
            stroke: #8F49FE;
          }
        };

      @keyframes trackedAnimation {
        from {
          stroke-dasharray: 100;
          stroke-dashoffset: 100;
        }

        to {
          stroke-dasharray: 100;
          stroke-dashoffset: 20;
        }
        };

        @keyframes untrackedAnimation {
          0% {
            transform: rotate(0deg);
            stroke-dasharray: 100;
          }

          40% {
            stroke-dasharray: 80;
          }

          80% {
            stroke-dasharray: 100;
          }

          100% {
            transform: rotate(360deg);
          }
        };
```